wiki/Tournaments/CC/6/en.md

```diff
@@ -0,0 +1,510 @@
+---
+tags:
+  - CC
+  - CC6
+  - CCVI
+  - CC 6
+  - CC VI
+  - CC#6
+---
+
+# Corgi Cup VI
+
+![CCVI logo](img/logo.png)
+
+**Corgi Cup VI** (***CCVI***) was a country-based double-elimination osu! tournament hosted by ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), and ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the sixth installment of the Corgi Cup.
```

Comment:
Shouldn't this be `**Corgi Cup VI** (***CCVI***) was a 1v1 double-elimination osu! tournament hosted by ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), and ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the sixth installment of the Corgi Cup.`

Reply:
I guess I'll just change this when pulling in the next PR to save dealing with reopening this.